libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -282,6 +281,11 @@ pub fn crop_ballot_page_image_borders(mut image: GrayImage) -> Option<BallotImag
         )
         .to_image();
 
+    // Re-compute the threshold after cropping to ensure future
+    // re-interpretations based on the saved image are consistent with the
+    // initial one.
+    let threshold = otsu_level(&image);
```

Comment:
Nice! Could having to compute this twice have a meaningful effect on interpretation speed? Not sure how slow/fast this operation is

Reply:
It is non-negligible, but still a fairly small absolute number. On my M4 MBP it's 6ms. On the production machines I'm betting it's at worst 10x that, or 60ms.

Reply:
Awesome, yeah that's totally reasonable